README.md

```diff
@@ -50,6 +50,6 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 [MIT](https://choosealicense.com/licenses/mit/)
 
 ---
-This project is part of the [Swarms](https://swarms.com/) initiative 🚀
+This project is part of the [Swarms](https://swarms.world/) initiative 🚀
 
-![Swarms Logo](swarms_logo.png)
\ No newline at end of file
+![Swarms Logo](swarms_logo.png)
```

Comment:
@evelynmitchell  - I've looked at line 55 different ways with different tools and don't see the whitespace - where is the whitespace change?  At the end of the line?

Can you let me know what tools you are using to see this?

Reply:
Its at the end of the line. 

I use vscode configured with markdownlint to spot whitespace changes.

This PR looks good to me.